src/sentry/constants.py

```diff
@@ -64,7 +64,7 @@ def get_all_languages() -> List[str]:
 # accuracy provided.
 MINUTE_NORMALIZATION = 15
 
-MAX_TAG_KEY_LENGTH = 32
+MAX_TAG_KEY_LENGTH = 200
```

Comment:
I think we can actually remove this constant, it does not seem to be read anywhere.

```suggestion
```

Reply:
Agreed.